### ISSUE TITLE
`import json` is included in Tasks that don't need it

### ISSUE THREAD

**Issue by dmerrick** [argoproj-labs/hera-workflows#156]
Right now Hera adds some json boilerplate to its definitions. As [the code mentions](https://github.com/argoproj-labs/hera-workflows/blob/main/src/hera/task.py#L640), intended to be a convenience feature, and it is useful:

> Since Argo passes parameters through {{input.parameters.name}} it can be very cumbersome for users to manage that. This creates a script that automatically imports json and loads/adds code to interpret each independent argument into the script.

But there are situations where adding this import is unnecessary:
* python functions that don't use input parameters or `json`
* non-python functions like [the `cowsay` example](https://github.com/argoproj-labs/hera-workflows/blob/main/examples/container.py#L10)
* situations where workflow filesize needs to be minimized

Here's an example snippit from a Workflow created by hera:
```
      script:
        ...
        command:
          - bash
          - '-c'
          - argo list
        ...
        source: |+
          import json
```

**Comment by flaviuvadan** [comment]
Hey @dmerrick! 👋🏻  I agree with your assessment - we can definitely remote the unnecessary source script when a script is not actually passes! Would you like to submit a PR? 🙂

**Comment by dmerrick** [comment]
I was going to and then I realized I wasn't 100% sure how to do so without potentially breaking things, so I ended up creating this issue instead :)

I might revisit this as part of my work related to https://github.com/argoproj-labs/hera-workflows/issues/151

**Comment by flaviuvadan** [comment]
@dmerrick a fix is coming in 3.3.0

**Comment by dmerrick** [comment]
Awesome!
